### PR TITLE
doc/user: add release notes for v0.2.1

### PR DESCRIPTION
**Pull request by benesch**
This change is [Reviewable](https://reviewable.io/reviews/materializeinc/materialize/2831)

### CODE REVIEW

**Comment by quodlibetor** [comment]
cc #2722

File: doc/user/release-notes.md [hunk 1]
```diff
@@ -52,7 +52,28 @@ Wrap your release notes at the 80 character mark.
   UPSERT  ...`](../overview/api-components/#envelopes))
 
 <span id="0.2.1"></span>
-## 0.2.0 &rarr; 0.2.1 (Unreleased)
+## 0.2.0 &rarr; 0.2.1
+
+- Allow [`EXPLAIN`](../sql/explain) statements to take parameters (`$1`, `$2`,
```
Comment (sploiselle) [inline doc/user/release-notes.md:57]: I am/was a little confused by this: probably not a big deal but is "Support `EXPLAIN` for statements that contain parameters..." also correct?

Comment (benesch) [reply doc/user/release-notes.md]: Ah, yeah, it is a little weirdly worded. I rephrased to "Allow query parameters (`$1`, `$2`, etc) to appear in EXPLAIN statements." How does that sound? (Something about your suggestion triggered I think the same level of low-grade parser confusion in my brain.)

Comment (sploiselle) [reply doc/user/release-notes.md]: LGTM

**Pull request merged**